backend/tests/manual_upload_test.py

```
@@ -0,0 +1,395 @@
+1. Generating test files
+2. Testing them against the upload endpoint
+3. Providing detailed reports
+
+Usage:
+    python manual_upload_test.py [--server-url http://localhost:8000] [--output-dir ./test_files]
+"""
+
+import argparse
+import requests
+import os
+import tempfile
```

Ruff (0.11.9)

`tempfile` imported but unused

Remove unused import: `tempfile`

(F401)